src/NLog/Layouts/SimpleLayout.cs

```diff
@@ -263,14 +263,15 @@ protected override string GetFormattedMessage(LogEventInfo logEvent)
                 }
                 catch (Exception exception)
                 {
-                    if (exception.MustBeRethrown())
+                    //also check IsErrorEnabled, otherwise 'MustBeRethrown' writes it to Error
+                    if (InternalLogger.IsWarnEnabled || InternalLogger.IsErrorEnabled)
```

Comment:
I mean I think the entire condition `if (InternalLogger.IsWarnEnabled || InternalLogger.IsErrorEnabled)` can be removed. `InternalLogger.Warn()` will not actually log anything if the LogLevel is lower than `Warn`, right?